docs/source/user_documentation/user_documentation.rst

```diff
@@ -125,6 +125,21 @@ SnowShu will report details of the created replica once completed.
 
 .. image:: /../assets/completed_replica.png 
 
+Creating An Incremental Replica
+------------------
+
+You have already built a replica but made some changes in ``replica.yml`` file. Instead of building a new replica, you can apply changes to the existing replica.
```

Comment:
The first sentence seems incomplete to me - maybe rephrase it or end with question mark?